shapeshift/web
# fix: keepkey swaps from RUNE

## Description

Bumps `chain-adapters` to latest, see https://github.com/shapeshift/lib/pull/1231 for more info on what this does and why.

tl;dr uses a `MsgDeposit` message that `hdwallet-keepkey` accepts and won't throw, and that can be signed by the KeepKey firmware, making swaps from RUNE on KeepKey great (unfortunately not again).

## Notice




## Pull Request Type


## Issue (if applicable)

N/A

## Risk



None, can't break what's already broken

## Testing



- Connect a KeepKey
- Ensure THORChain swaps from RUNE are working
- As a sanity check, ensure swaps to RUNE are also working, though the lib PR doesn't actually touch this

- Expected/min. amounts aren't a concern of this PR

### Engineering



- ☝🏽 

### Operations



- ☝🏽 

## Screenshots (if applicable)

<img width="426" alt="image" src="https://user-images.githubusercontent.com/17035424/229823239-e7a7d844-9885-4ca6-ade6-55af4ab58049.png">
<img width="430" alt="image" src="https://user-images.githubusercontent.com/17035424/229823293-2ff8f675-c696-4c07-917b-dd1e112c8e75.png">
<img width="424" alt="image" src="https://user-images.githubusercontent.com/17035424/229823582-38f09ee5-8d8f-4174-a4c0-2038493ff640.png">
<img width="419" alt="image" src="https://user-images.githubusercontent.com/17035424/229823752-bcd1308d-1c01-4be4-a4fc-1d2d1ddd3d0a.png">
<img width="1216" alt="image" src="https://user-images.githubusercontent.com/17035424/229823860-1a5d10e2-3726-4711-84d5-36fa27e70b59.png">
